nixos/modules/system/boot/systemd/initrd.nix

```diff
@@ -428,8 +428,8 @@ in {
         "${cfg.package}/lib/cryptsetup/libcryptsetup-token-systemd-fido2.so"
         "${pkgs.libfido2}/lib/libfido2.so.1"
 
-        # the unwrapped systemd-cryptsetup executable
-        "${cfg.package}/lib/systemd/.systemd-cryptsetup-wrapped"
+        # the systemd-cryptsetup executable
+        "${cfg.package}/lib/systemd/systemd-cryptsetup"
```

Comment:
It's definitely weird that we had the unwrapped one here and the wrapped one in `luksroot.nix`. But maybe we should leave the wrapped one in `luksroot.nix` and simply remove the line from this file? I'm pretty sure we don't want this in initrd unless LUKS is actually being used.

Reply:
It looks like there is a reference to the exe that could get dynamically added from systemd-gpt-auto-generator, but that's because the discoverable partitions spec does allow discovery of LUKS partitions and it's probably not important to have work if LUKS devices aren't defined in the config (someone can just `boot.initrd.luks.forceLuksSupportInInitrd` anyway to get it even if the device itself isn't in the config)

I'll push an update in a second

Reply:
I've no clue why it was added to begin with though so 🤷🏻‍♀️

Reply:
It looks like it came from https://github.com/NixOS/nixpkgs/pull/189676. I assume because they ran into the wrapped issue but they added it to the systemd-initrd module instead of the LUKS one for some reason

@zhaofengli Was there a reason for adding the wrapped executable here or can it be removed now that make-initrd-ng supports wrapped files with this PR?